data/remnant ships.txt

```diff
@@ -560,3 +696,62 @@ ship "Starling" "Starling (Thrasher)"
 	gun
 
 
+
+ship "Tern"
+	sprite "ship/tern"
+	thumbnail "thumbnail/tern"
+	attributes
+		category "Drone"
+		licenses
+			Remnant
+		"cost" 3830000
+		"shields" 1000
+		"hull" 4000
+		"cargo space" 10
+		"fuel capacity" 100
+		"ramscoop" 1
+		"automaton" 1
+		"mass" 30
+		"drag" .9
+		"heat dissipation" .9
+		"gaslining" 1
+		"outfit space" 111
+		"shield generation" 1.8
+		"shield energy" 1.1
+		"hull repair rate" 0.7
+		"hull energy" 0.6
+		"energy generation" 1.8
+		"heat generation" 3.0
+		"atmosphere scan" 100
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		"asteroid scan power" 30
+		"weapon capacity" 36
+		"engine capacity" 40
+		weapon
+			"blast radius" 10
+			"shield damage" 100
+			"hull damage" 150
+			"hit force" 250
+	outfits
+		"Inhibitor Cannon" 2
+		
+		"Millennium Cell"
+		"Crystal Capacitor"
+		"Thermoelectric Cooler"
+		
+		"Crucible-Class Thruster"
+		"Crucible-Class Steering"
+		
+	gun -5 -20 "Inhibitor Cannon"
+	gun 5 -20 "Inhibitor Cannon"
+	turret 0 -8
+	engine -5 20.5
+	engine 5 20.5
+	explode "tiny explosion" 30
+	explode "small explosion" 20
+	description "The Tern is a small, hardy, drone developed to plunge into the atmosphere of nearby gas giants to collect valuable resources for the nearby Remnant worlds. As a result, these tough, thick shelled craft are surprisingly effective at providing distraction and covering fire when enemies intrude on Remnant space."
```

Comment:
`small, hardy, drone` sounds off to me, I'm no grammer nanna but shouldn't it be:
`small, hardy drone`

Reply:
I'm inclined to agree with you. It does look better without it.

Fixed.